apps/dashboard/src/model-assessment-vision/__mock_data__/fridgeObjectDetection.ts

```diff
@@ -10,6 +10,54 @@ export const fridgeObjectDetection: IDataset = {
   class_names: ["can", "carton", "milk_bottle", "water_bottle"],
```

Comment:
can you reduce the size of the file changes here?
[fridgeObjectDetection.ts](https://github.com/microsoft/responsible-ai-toolbox/pull/2010/files#diff-f752661a8a82493c00cc75d5da78f7bec027372ba91b5e4e66ece7bd4493af3b)
it looks like there are 1325 lines inserted - for these test apps I think we should just stay to a couple rows, this seems too much data for the test app

Reply:
fixed, makes sense!